src/state/accountState.ts

```diff
@@ -0,0 +1,76 @@
+import { EtherscanProvider, JsonRpcProvider, StaticJsonRpcProvider, Web3Provider } from "@ethersproject/providers";
+import { createSlice, createSelector, createAsyncThunk } from "@reduxjs/toolkit";
+import { BigNumber, ethers } from "ethers";
+import { SupportedChainId } from "../constants/chains";
+import { RootState } from "./store";
+import { tokens } from "../contracts/contracts";
+import { GOHM } from "../typechain";
```

Comment:
I realized that we don't need gohm specific ABIs. I will change this to be a standard ERC20 ABI.